src/components/tx-flow/common/TxLayout/styles.module.css

```diff
@@ -1,5 +1,5 @@
 .container {
-  margin-top: 42px;
+  margin-top: 10px;
```

Comment:
The modal alignment no longer matches:

![tx-modal-jump](https://github.com/safe-global/safe-wallet-web/assets/20442784/b01ebfb7-5b10-4782-9983-14a1782c8ba8)